src/main/jbake/content/docs/components/btm/index.adoc

```diff
@@ -47,26 +59,29 @@ Instructions on how to configure the client environment can be found here: link:
 
 === The Server
 
-The Business Transaction Management (BTM) Server supports reporting and analysis of the business transaction data. These capabilities can be access through a set of REST services, more information can be found here: link:../../rest/rest-btm.html[BTM REST services]
+The Hawkular BTM Server supports reporting and analysis of the business transaction data. These capabilities can be accessed through a set of REST services, more information can be found here: link:../../rest/rest-btm.html[BTM REST services]
 
 The instructions for installing the server can be found here: link:serverinstall.html[Server Installation]
 
 === The User Interface
 
-The BTM User Interface provides the capability to:
+The Hawkular BTM User Interface provides the capability to:
 
-* Configure business transactions
+* View Application Performance information
 
-* View a list of business transactions, with summary statistics
+* View Distributed Tracing
 
-* Access more detailed analytics for the business transactions
+* Manage Business Transactions
+** Configure business transaction definitions
+** View a list of business transactions, with summary statistics
+** Access more detailed analytics for the business transactions
 
 These capabilities will be discussed in more detail in the following section.
 
 
 == Console
```

Comment:
I think it would be good for each of the sections to have a screen shot with annotations that are then in the text below referenced to
E.g. instead of "On the left hand side below the graph is an area for specifying filters" having that area in the screenshot marked with (1) and then writing   "1) This is an area for specifying filters..."